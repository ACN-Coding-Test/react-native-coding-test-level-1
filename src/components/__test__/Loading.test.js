import React from 'react';
import renderer from 'react-test-renderer';
import {Loading, Button, DateInput, TextInput} from '..';

test('renders Loading correctly', () => {
  const tree = renderer.create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});
