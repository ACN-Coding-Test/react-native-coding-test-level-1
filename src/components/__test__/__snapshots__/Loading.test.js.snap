// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Loading correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
  testID="loading"
>
  <ActivityIndicator
    color="#39559D"
    size="large"
  />
</View>
`;
